content/04.discussion.md

```diff
@@ -52,3 +52,15 @@ These requirements make the application of dominating set differential abundance
 + assembly graphs can be built with a combination of long and short reads or from long or short reads alone, so dominating set differential abundance analysis is a flexible method to compare groups of metagenomes that will accommodate the field as metagenome sequencing continues to improve
 + the framework is theoretically extensible to metatranscriptomes, but this remains a point of future research
 -->
+
+While we present an initial pipeline that enables differential abundance analysis directly on assembly graphs, we identified several areas where our approach could be improved.
+First, implementing approaches to better control graph piece size would be beneficial.
+We heuristically selected *r* = 10 to build the metapangeome graphs because this radius size produced dominating set pieces with approximately the same number of k-mers as the average bacterial gene for pieces that were present in many metagenomes.
+However, the radius size needed to meet this condition may change depending on the diversity of the sequencing reads used to build the metapangenome graph.
+Diversity increases with the complexity of the sequenced community, sequencing depth, and the number of communities observed.
+Algorithms that either automatically select a radius size that achieves a user-specified average piece size, or that produce more consistent piece sizes independent of diversity of the sequencing data would provide finer control of the graph structure and subsequent sequence comparisons.
+Switching from a cDBG to a de Bruijn graph as the base spacegraphcats graph structure could lead to more consistent piece sizes; cDBGs have variable node sizes because they combine nodes without branching paths, while every node in a de Bruijn graph is length *k*. 
```

Comment:
Maybe "Further, switching..." ?

Reply:
```suggestion
Further, switching from a cDBG to a de Bruijn graph as the base spacegraphcats graph structure could lead to more consistent piece sizes; cDBGs have variable node sizes because they combine nodes without branching paths, while every node in a de Bruijn graph is length *k*. 
```